src/js/components/stock-movement-wizard/outboundImport/OutboundImport.jsx

```diff
@@ -73,9 +77,19 @@ const OutboundImport = () => {
     <PageWrapper>
       <WizardStepsV2 steps={stepsTitles} currentStepKey={Step.key} />
       <OutboundImportHeader />
-      <form onSubmit={handleSubmit(onSubmit)}>
+      <form onSubmit={handleSubmit(onSubmitStockMovementDetails)}>
         {is(OutboundImportStep.DETAILS.key) && (<Step.Component {...detailsComponentProps} />)}
-        {is(OutboundImportStep.CONFIRM.key) && (<Step.Component previous={previous} />)}
+      </form>
+      <form onSubmit={handleSubmit(onConfirmImport)}>
```

Comment:
nice idea with having two separate forms, triggering the same `handleSubmit` - I assume we will be able easily to manipulate whether we trigger the validation endpoint or the save (onSubmitStockMovementDetails would trigger the validation endpoint and the onConfirmImport would trigger the save)